modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -143,6 +145,24 @@ void AudioStreamPlaybackMP3::tag_used_streams() {
 	mp3_stream->tag_used(get_playback_position());
 }
 
+void AudioStreamPlaybackMP3::set_parameter(const StringName &p_name, const Variant &p_value) {
+	if (p_name == SNAME("looping")) {
+		if (p_value == Variant()) {
+			looping_override = false;
+			looping = false;
+		} else {
+			looping_override = true;
+			looping = p_value;
+		}
+	}
+}
+Variant AudioStreamPlaybackMP3::get_parameter(const StringName &p_name) const {
+	if (p_name == SNAME("looping") && looping_override) {
```

Comment:
```suggestion
	if (looping_override && p_name == SNAME("looping")) {
```
Cheaper check first.
Same in OGG.